### PR TITLE
quickstart-tweaks

### PR DESCRIPTION
**Pull request by billpalombi**
- Ends all sentences proceeding code/command blocks with a colon.
- Minor clarifications/additions to the converting to a workflow section

### CODE REVIEW

File: docs/3.0rc/get-started/quickstart.mdx [hunk 1]
```diff
@@ -118,7 +119,7 @@ The `log_prints=True` argument provided to the `@flow` decorator logs output fro
 
 ## Run your flow
 
-Run your Prefect flow.
+Run your Prefect flow just as your would a python script:
```
Comment (discdiver) [inline docs/3.0rc/get-started/quickstart.mdx:122]:
```suggestion
Run your Prefect flow just as your would a Python script:
```

**Review by discdiver — COMMENTED** [review]
One nit.

File: docs/3.0rc/get-started/quickstart.mdx [hunk 2]
```diff
@@ -68,10 +68,11 @@ If you have any issues with browser-based authentication, see the [Prefect Cloud
 
 ## Convert your script to a Prefect flow
 
-The easiest way to convert a Python script into a workflow is to add a `@flow` decorator.
-[Flows](/3.0rc/develop/write-flows/) are the core observable, deployable units in Prefect and are the primary entrypoint to orchestrated work.
+The easiest way to convert a Python script into a workflow is to add a `@flow` decorator to the script's entrypoint, the Python function that runs first.
+[Flows](/3.0rc/develop/write-flows/) are containers for workflow logic as code. 
+They core observable, orchestrated, deployable units of work in Prefect.
```
Comment (billpalombi) [inline docs/3.0rc/get-started/quickstart.mdx:73]:
```suggestion
They're the core observable, orchestrated, deployable units of work in Prefect.
```

File: docs/3.0rc/get-started/quickstart.mdx [hunk 3]
```diff
@@ -68,10 +68,11 @@ If you have any issues with browser-based authentication, see the [Prefect Cloud
...
 
-Convert the script into a flow with two tasks, each defined by the `@task` decorator.
+You can also add `@task` decorators to the functions called by the flow to track and orchestrate them.
```
Comment (billpalombi) [inline docs/3.0rc/get-started/quickstart.mdx:75]:
```suggestion
Adding `@task` decorators to the functions called by the flow converts them to tasks. 
Tasks receive metadata about upstream dependencies and the state of those dependencies before they run.
Prefect will record these dependencies and states as it orchestrates these tasks when they run.
```

File: docs/3.0rc/get-started/quickstart.mdx [hunk 4]
```diff
@@ -68,10 +68,11 @@ If you have any issues with browser-based authentication, see the [Prefect Cloud
...
 
-The easiest way to convert a Python script into a workflow is to add a `@flow` decorator.
-[Flows](/3.0rc/develop/write-flows/) are the core observable, deployable units in Prefect and are the primary entrypoint to orchestrated work.
+The easiest way to convert a Python script into a workflow is to add a `@flow` decorator to the script's entrypoint, the Python function that runs first.
+[Flows](/3.0rc/develop/write-flows/) are containers for workflow logic as code. 
+They core observable, orchestrated, deployable units of work in Prefect.
 
-Convert the script into a flow with two tasks, each defined by the `@task` decorator.
+You can also add `@task` decorators to the functions called by the flow to track and orchestrate them. 
 Tasks are the smallest unit of observed and orchestrated work in Prefect.
```
Comment (billpalombi) [inline docs/3.0rc/get-started/quickstart.mdx:76]:
```suggestion
```

**Pull request merged**